src/patternfly/components/Wizard/wizard.scss

```diff
@@ -538,7 +538,9 @@
 
 .#{$wizard}__main {
   z-index: var(--#{$wizard}__main--ZIndex);
+  display: flex;
   flex: 1 1 auto;
```

Comment:
I think you'll want to remove this line as default `flex` values resolve to:

```
flex-grow: 0;
flex-shrink: 1;
flex-basis: auto;
```

And all you need to update is `:last-child` if that's an agreeable solution.

Reply:
At least for the flex property that should still be needed so that the wizard main element fills whatever space to match the current behavior when a drawer is rendered inside wizard main.

@mcoker can't recall what the discussion was, but was flex-grow on the `:last-child` necessary? Believe testing these styles in my React PR omitting the flex-grow on last child doesn't change anything (though i may just not be using the combo of elements to notice a difference)